js_upload/js_upload.php

```diff
@@ -106,22 +106,22 @@ function createUploader() {
 
 			if (typeof acl != "undefined"){
 				uploader.setParams( {
-					newalbum      : newalbum,
-					album         : album,
+					newalbum	  : newalbum,
+					album		 : album,
```

Comment:
Please revert these alignment change that must remain spaces.

Reply:
I already reverted it with https://github.com/friendica/friendica-addons/pull/827/commits/e0815309d0812d5b4dc5fa8ba7b9195712ee272b